assets/src/dashboard/app/views/storyAnimTool/timeline/index.js

```diff
@@ -213,7 +213,10 @@ function Timeline({
 
       onAddOrUpdateAnimation(
         Object.keys(animation)
-          .filter((name) => typeof animation[name] !== 'undefined')
+          .filter(
+            (name) =>
+              typeof animation[name] !== 'undefined' && animation[name] !== ''
+          )
           .reduce(
```

Comment:
This removes uninitialized properties from being passed to the animationPart logic.  I was getting errors that an empty string wasn't a valid value for `easing`, but this fixes it.

Reply:
oh yea, I did a quick and dirty fix for this in the scrubbing feature, but I like this better.

Also may be nice to make easing a dropdown with an option for custom input cus I find myself referencing these a lot:
https://easings.net/#easeOutSine